create/create.js

```diff
@@ -0,0 +1,23 @@
+import { createNewPost, checkAuth } from '../fetch.utils.js';
+
+const form = document.getElementById('create-post');
+const homeBtn = document.getElementById('home-btn');
+
+checkAuth();
+
+form.addEventListener('submit', async (e) => {
+    e.preventDefault();
+    const data = new FormData(form);
+    const newPost = {
+        title: data.get('title'),
+        description: data.get('description'),
+        contact: data.get('contact'),
+    };
+    const response = await createNewPost(newPost);
+    // console.log(response);
```

Comment:
Your posts are rendering twice... it might be because you create a variable of 'response' but don't ever return it? Let's look at it together!